### ISSUE TITLE
Put a confirm dialog box when pressing "reset settings" button

### ISSUE THREAD

**Issue by denielton** [wwmm/easyeffects#2273]
### EasyEffects Version

7.0.3-1

### What package are you using?

Arch (easyeffects)

### Distribution

Manjaro

### Describe the bug

Easyeffects delete all presets files and restore default settings when clicking "reset settings" without a dialog box asking to confirm the action.

### Expected Behavior

Dialog box appearing when pressing "reset settings" with the text "do you really want to delete all your preset config files and restore default settings? this action can not be undone"

### Debug Log

<details><summary>Debug Log</summary><br/>

```
Paste your log here
```
</details>

### Additional Information

please

**Comment by wwmm** [comment]
>Easyeffects delete all presets files

No. It doesn't delete the user preset files as well as the impulse response files. They stay inside `~/.config/easyeffects`. What is in its gsettings database that is cleared.

**Comment by DomiStyle** [comment]
+1 for this. I accidentally hit it as well and lost my active effect pipeline.

Is there a way to restore it? I still have a btrfs snapshot from before I reset it but I can't find where EasyEffects stores those.

**Comment by DomiStyle** [comment]
Figured it out, the gsettings database is stored in `~/.var/app/com.github.wwmm.easyeffects/config/glib-2.0/settings/keyfile`.
Restoring that file from backup did the trick.

**Comment by wwmm** [comment]
The 8.x releases show a confirmation dialog when the reset all settings menu entry is activated.
